bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -3008,7 +3008,11 @@ void ReplicaImp::MoveToHigherView(ViewNum nextView) {
       if (seqNumInfo.getPrePrepareMsg() != nullptr && seqNumInfo.isTimeCorrect()) {
         ViewsManager::PrevViewInfo x;
 
-        std::tie(x.prePrepare, x.prepareFull) = seqNumInfo.getAndReset();
+        PrepareFullMsg *fullMsg;
+        PrePrepareMsg *partialMsg;
+        std::tie(partialMsg, fullMsg) = seqNumInfo.getAndReset();
```

Comment:
why not using structured binding?
`auto [partialMsg, fullMsg] = seqNumInfo.getAndReset();`

Reply:
Done